### PR TITLE
docs(PPDSC-2346): Grid cell documentation

### PR DESCRIPTION
**Pull request by RashikaNewsUK**
PPDSC-2346

**What**

1.Grid-Cell documentation UI.

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [X] Updated relevant documentation

**I have tested manually:**
- [X] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [X] The screen reader reads and flows through the elements as expected.
- [X] There are no new errors in the browser console coming from this PR.
- [X] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2346-grid-cell-documentation
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2346-grid-cell-documentation/storybook

File: site/pages/components/grid.tsx [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+            },
+          ],
+          overridesRows: commonOverridesRows,
+        },
+        {
+          title: 'Cell',
+          summary: `The cell has a range of props that can be used to define an appropriate experience for different use cases.`,
+          propsRows: [
+            {
+              name: 'Root',
```
Comment (mutebg) [inline site/pages/components/grid.tsx:375]:
```suggestion
              name: 'children',
```

File: site/pages/components/grid.tsx [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+                'smHidden',
+                'mdHidden',
+                'lgHidden',
+                'xlHidden',
+              ],
+              default: '',
+              description:
+                'The children of a column are rendered in that column, any supported React node is allowed.',
+              required: true,
+            },
```
Comment (mutebg) [inline site/pages/components/grid.tsx:395]: Tihs does not seems correct, we have the following props:
```
  xs?: number | 'full-width';
  sm?: number | 'full-width';
  md?: number | 'full-width';
  lg?: number | 'full-width';
  xl?: number | 'full-width';

  xsHidden?: boolean;
  smHidden?: boolean;
  mdHidden?: boolean;
  lgHidden?: boolean;
  xlHidden?: boolean;

  xsOrder?: number;
  smOrder?: number;
  mdOrder?: number;
  lgOrder?: number;
  xlOrder?: number;

  xsOffset?: number;
  smOffset?: number;
  mdOffset?: number;
  lgOffset?: number;
  xlOffset?: number;
```

**Review by mutebg — CHANGES_REQUESTED** [review]

File: site/pages/components/grid.tsx (hunk 2 above)
Comment (RashikaNewsUK) [reply site/pages/components/grid.tsx]: Hi @mutebg ,
All the changes has been done. Please have a look on it.
Thanks,
Rashika

File: site/pages/components/grid.tsx [hunk 3]
```diff
@@ -8,97 +9,220 @@ import {Link} from '../../components/link';
...
+
+const infoIcon = (
+  <IconFilledInfo
+    overrides={{
+      size: 'iconSize020',
+    }}
+  />
+);
+
+const commonPropsRows = [
```
Comment (mutebg) [inline site/pages/components/grid.tsx:22]:
```suggestion
const gridPropsRows = [
```

File: site/pages/components/grid.tsx [hunk 4]
```diff
@@ -356,42 +480,328 @@ const GridCellComponent = (layoutProps: LayoutProps) => (
...
-          overridesRows: commonOverridesRows,
+          propsRows: commonPropsRows,
         },
         {
           title: 'Cell',
           summary: `The cell has a range of props that can be used to define an appropriate experience for different use cases.`,
           propsRows: [
             {
-              name: 'Root',
+              name: 'Children',
```
[the remark this answers is not in the document]
Comment (RashikaNewsUK) [reply site/pages/components/grid.tsx]: Hi @mutebg , 
changes has been done, Please have a look on it. -thanks!

**Review by mutebg — APPROVED** [review]

**Review by Srinivasan-Ramasamy — APPROVED** [review]

**Pull request merged**